src/integrationTest/java/uk/gov/hmcts/reform/hois/controllers/HoisControllerTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.hois.controllers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.hois.domain.entities.HomeOfficeRequest;
+
+@WebMvcTest
```

Comment:
Probably better to use something like 
```@RunWith(SpringRunner.class)
@SpringBootTest(classes = Application.class, webEnvironment = MOCK)```

See ia-case-api for reference

Reply:
Do we need the Spring context to be set up for web test?

Reply:
Changed to SpringBootTest